lib/docker/artifact_image.js

```diff
@@ -70,32 +70,37 @@ export default class ArtifactImage {
 
     let originalTarball = path.join(downloadDir, 'image.tar');
 
-    let newTarball;
     try {
-      await downloadArtifact(
-        this.queue,
-        this.stream,
-        this.taskId,
-        this.artifactPath,
-        originalTarball,
-        this.runtime.dockerConfig
+      await this.runtime.stats.timeGen('taskImageDownloadTime',
```

Comment:
That's a completely fair point; I agree!.  It's something that's been done in docker-worker for awhile now, and not for a really good reason.

This wouldn't be something that we would want to continue doing in new workers.